tests/test_topy.py

```diff
@@ -0,0 +1,168 @@
+import os
+import time
+
+import ansys.fluent.core as pyfluent
+
+
+def test_single_jou(with_launching_container):
+
+    file_path = os.path.join(pyfluent.EXAMPLES_PATH, "jou1.jou")
+
+    with open(file_path, "w") as journal:
+        journal.write('(display "from jou1.jou")')
+
+    solver = pyfluent.launch_fluent(mode="solver", topy=file_path)
+    solver.exit()
+
+    with open(file_path) as file:
+        returned = file.readlines()
+
+    if os.path.exists(file_path):
+        os.remove(file_path)
+
+    assert returned
+
+    gen_file_path = f'{file_path.split(".")[0]}.py'
+
+    with open(gen_file_path) as file:
+        gen_returned = file.readlines()
+
+    assert gen_returned
+
+    while os.path.exists(gen_file_path):
```

Comment:
We should always make such loops finite even when we know that it's not necessary.

Reply:
Yes, sure. Thank you.

Reply:
Done. Thank you.